test-apps/display-performance-test-app/src/frontend/TestRunner.ts

```diff
@@ -1028,51 +1037,50 @@ function getRenderOpts(curRenderOpts: RenderSystem.Options): string {
           }
         }
         break;
-      case "preserveShaderSourceCode":
-        if (value) optString += "+shadeSrc";
-        break;
+      }
       case "displaySolarShadows":
-        if (!value) optString += "-solShd";
+        if (!curRenderOpts[key]) optString += "-solShd";
         break;
-      case "logarithmicZBuffer":
```

Comment:
@DStradley this was always a bug - wrong property name - compiler caught it now. Inverted this and some others because they are on by default.